pelicanconf.py

```diff
@@ -18,12 +18,8 @@
 DEFAULT_PAGINATION = False
 
 THEME = "theme"
-INDEX_SAVE_AS = 'news.html'
+INDEX_SAVE_AS = 'index.html'
```

Comment:
if you do that you'll replace the landing page by something else (the list of articles here).